src/game/server/entities/character.h

```diff
@@ -134,7 +136,7 @@ class CCharacter : public CEntity
 
 	// DDrace
 
-
+	int m_NeededFaketuning; // TODO: DDNet7 implement this https://github.com/ddnet/ddnet/commit/92e0b5fe3c0471ae5e2a71e7a6794fb7df71380a
```

Comment:
No, for sure not.

Reply:
Well then lets see it as a placeholder for 0.6 compatibility for now.